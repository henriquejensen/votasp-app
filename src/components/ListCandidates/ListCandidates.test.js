import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import firebase from 'firebase/app';

import ListCandidates from './ListCandidates';

describe('ListCandidates', () => {
  it('should match a snapshot', () => {
    const get = () => Promise.resolve();
    const where = jest.fn().mockReturnValue({ get });
    const collection = jest.fn().mockReturnValue({ where });
    firebase.firestore = jest.fn().mockReturnValue({ collection });
    const Component = shallow(( <ListCandidates /> ))
    expect(toJson(Component)).toMatchSnapshot();

    Component.setState({ loading: false });
    expect(toJson(Component)).toMatchSnapshot();
  })
})