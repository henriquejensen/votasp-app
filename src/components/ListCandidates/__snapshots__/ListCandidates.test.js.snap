// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListCandidates should match a snapshot 1`] = `<Loader />`;

exports[`ListCandidates should match a snapshot 2`] = `
<PageLayout>
  <section
    className="container list-candidates"
  >
    <h3
      className="titulo"
    >
      Lista de Candidatos e Candidatas
    </h3>
    <h4>
      Veja abaixo a lista de candidatos e candidatas que já fizeram o cadastro ou estão com cadastro incompleto no Vota SP.
    </h4>
    <h4>
      Caso você procure o candidato ou candidata e não encontre é porque ele ou ela ainda não fez o cadastro.
    </h4>
    <h4>
      Agradecemos muito se puder avisá-lo ou avaisá-la para se cadastrar.
    </h4>
    <h4>
      Somente aparecem no ranking os cadastros completos
    </h4>
  </section>
</PageLayout>
`;
